bundles/CoreBundle/src/DependencyInjection/ConfigurationHelper.php

```diff
@@ -0,0 +1,107 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+use Symfony\Component\Config\Definition\Processor;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Finder\Finder;
+
+final class ConfigurationHelper
```

Comment:
This should be marked as `@internal` and maybe named a bit more specific, maybe `ConfigLocationHelper` ?

Reply:
@brush I also thought about a specific name but I put the function `getConfigNodeFromSymfonyTree` in this helper and maybe in the future we will need more helper functions which aren't related to the ConfigLocation. What do you think about that?